app-emulation/crun/crun-0.17.ebuild

```diff
@@ -14,20 +14,19 @@ SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz
 LICENSE="GPL-2+ LGPL-2.1+"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64"
-IUSE="+bpf +caps criu man +seccomp systemd static-libs"
+IUSE="+bpf +caps criu +seccomp systemd static-libs"
 
 DEPEND="
 	sys-kernel/linux-headers
 	>=dev-libs/yajl-2.0.0
 	caps? ( sys-libs/libcap )
-	criu? ( >=sys-process/criu-3.13 )
+	criu? ( >=sys-process/criu-3.15 )
 	seccomp? ( sys-libs/libseccomp )
 	systemd? ( sys-apps/systemd:= )
 "
 RDEPEND="${DEPEND}"
 BDEPEND="
```

Comment:
I would just remove BDEPEND

Reply:
do I not need PYTHON_DEPS there? according to https://dev.gentoo.org/~mgorny/python-guide/any.html I do...

the build system depends on it:
https://github.com/containers/crun/blob/0.17/configure.ac#L21